webapp-django/crashstats/crashstats/views.py

```diff
@@ -136,6 +139,79 @@ def build_data_object_for_crash_reports(response_items):
     return crash_reports
 
 
+def get_report_list_from_super_search(**kwargs):
+    '''Return results as expected by the report_list views, but loaded
+    from Super Search.
+
+    This function converts given parameters to match what Super Search expects.
+    It changes some values (like version and _sort parameters). It then
+    queries Super Search, and maps the field names in the results to what
+    the report/list/ views and templates expect.
+    '''
+    # Prepare mapping of names to in_database_names.
+    fields = SuperSearchFields().get()
+    old_names_map = {}
+    new_names_map = {}
+    for field in fields.values():
+        if (
+            (field['is_exposed'] or field['is_returned']) and
+            field['name'] != field['in_database_name']
+        ):
+            old_names_map[field['in_database_name']] = field['name']
+            new_names_map[field['name']] = field['in_database_name']
+
+    # Convert parameters to what Super Search accepts.
+    params = kwargs.copy()
+    if 'signature' in params:
+        params['signature'] = '=' + params['signature']
+
+    if 'end_date' in params or 'start_date' in params:
+        params['date'] = []
+        if 'start_date' in params:
+            start_date = params['start_date'].isoformat()
+            params['date'].append('>=' + start_date)
+            del params['start_date']
+        if 'end_date' in params:
+            end_date = params['end_date'].isoformat()
+            params['date'].append('<=' + end_date)
+            del params['end_date']
+
+    if 'version' in params:
+        versions = [
+            x.split(':')[-1] for x in params['version'] if x != 'ALL:ALL'
+        ]
+        params['version'] = versions
+
+    if 'process_type' in params and params['process_type'] == 'any':
+        del params['process_type']
+
+    if 'hang_type' in params and params['hang_type'] == 'any':
+        del params['hang_type']
```

Comment:
In my last rebase, I added those 4 lines. It's removing the default ``any`` value for those 2 fields, because it's not understood by Super Search.